.github/workflows/release.yml

```
@@ -0,0 +1,21 @@
+name: Release
+
+on: [workflow_dispatch]
+
+jobs:
+  nuget:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository_owner == 'flutter-tizen' }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-dotnet@v1
```

It is recommended to use the latest version.

```suggestion
      - uses: actions/checkout@v3
      - uses: actions/setup-dotnet@v2
```

---

Fixed. Thanks!